src/emulator/app/include/app/discord.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#ifdef USE_DISCORD_RICH_PRESENCE
+#if DISCORD_RPC
```

Comment:
What is `DISCORD_RPC`?

Reply:
? this code is give by velocity, all is right and ready to merge